add_me_to_your_PATH/driver.yaml.template

```diff
@@ -2,9 +2,18 @@ apiVersion: batch/v1
 kind: Job
 metadata:
     name: ${RUNID}
+    labels:
+        git_repo: ${GIT_REPO}
+        git_branch: ${GIT_BRANCH}
 spec:
     ttlSecondsAfterFinished: 10
     template:
+        metadata:
+            annotations:
+                cluster-autoscaler.kubernetes.io/safe-to-evict: "false"
+            labels:
+                git_repo: ${GIT_REPO}
+                git_branch: ${GIT_BRANCH}
```

Comment:
We should make sure we use an unmodified branch name. It would probably be even more useful to include a `git_sha`:
https://github.com/beacon-biosignals/julia_pod/blob/763fa0c8e824bd9573819bdaa0dc628d7c3a2153/add_me_to_your_PATH/build_image#L91

Reply:
The use-case here is cleaning up after oneself when debugging; sometimes you want to select a very specific job, sometimes all the jobs you've been running recently.

But I guess having a third key-value pair with `git_sha` won't hurt, let me add that.

Reply:
My point here is that we should just provide the unmodified branch name here so that people could then use the label to look up details in the commit history. The modified branch name should just be used for the more restricted job names